stockflux-securities-master/src/components/SecuritiesTable.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import Components from "stockflux-components";
+import "./SecuritiesTable.css";
+
+// hard coded data to be replaced with data from backend
+let SECURITIES_DATA = [
+  { Exch: "LSE", Symbol: "AAPL", Name: "Apple inc." },
```

Comment:
I'm not sure if there's a convention on the front end but I generally find that it's better to use `camelCase` instead of `StudlyCase` for properties. If there's a convention for the project, then stick to that. It's probably something worth discussing with @JackG188

Reply:
Yep change made